### PR TITLE
Add requirements for Windows Collector

### PR DESCRIPTION
**Pull request by theletterf**
**Requirements**
- [X] The content follows Splunk guidelines for style and formatting.
- [X] There are no CLI errors when building the docs locally.
- [X] You are contributing original content.

**Describe the change**

Add a table of prerequisites to the main Windows docs for OTel Collector.

### CODE REVIEW

File: gdi/opentelemetry/deployments/deployments-windows-puppet.rst [hunk 1]
```diff
@@ -45,7 +39,7 @@ The class accepts the parameters described in the following table:
      - Description
      - Default value
    * - ``collector_version``
-     - Version of the Collector package to install, for example, ``0.25.0``. The version should correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
+     - Version of the Collector package to install, for example, ``0.25.0``. The version musts correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
```
Comment (pmcollins) [inline gdi/opentelemetry/deployments/deployments-windows-puppet.rst:42]:
```suggestion
     - Version of the Collector package to install, for example, ``0.25.0``. The version must correspond to :new-page:`Github Releases <https://github.com/signalfx/splunk-otel-collector/releases>` without the preceding ``v``. This attribute is required.
```

Comment (pmcollins) [reply gdi/opentelemetry/deployments/deployments-windows-puppet.rst:42]: Could also be `The version must correspond to a Github Release`, which may be more accurate.

**Review by pmcollins — COMMENTED** [review]
Just one note from me. @jeffreyc-splunk should probably take a look since he knows the most about this.

**Comment by jeffreyc-splunk** [comment]
The supported versions list looks good to me.

**Review by aurbiztondo-splunk — APPROVED** [review]

**Pull request merged**